browserstack/local.py

```diff
@@ -8,6 +8,13 @@ def __init__(self, key=None, binary_path=None, **kwargs):
     self.options = kwargs
     self.local_logfile_path = os.path.join(os.getcwd(), 'local.log')
 
+  def _get_version(self, a_path):
```

Comment:
This a file read on every init of the instance. 
check module versioning in python. 
Module versioning in python 
https://www.python.org/dev/peps/pep-0396/